modules/rosa_cluster/variables.tf

```diff
@@ -23,23 +18,23 @@ variable "availability_zones" {
 
 #Cluster settings
 variable "cluster_name" {
-    type = string
-    default = "kumudu-tf-01"
+  type    = string
+  default = "kumudu-tf-01"
 }
 
-variable rosa_openshift_version {
-    type = string
-    default = "4.11.35"
+variable "rosa_openshift_version" {
+  type    = string
+  default = "4.11.35"
 }
 
-variable account_role_prefix {
-    type = string
-    default = "kumudu"
+variable "account_role_prefix" {
+  type    = string
+  default = "kumudu"
 }
 
-variable operator_role_prefix {
-    type = string
-    default = "mobbkh"
+variable "operator_role_prefix" {
+  type    = string
+  default = "mobbkh"
 }
```

Comment:
Do we want to consider changing these to non-specific vars?

Reply:
Yes indeed